recipe/build.sh

```diff
@@ -72,20 +71,25 @@ elif [[ "${target_platform}" == linux-* ]]; then
     sed -i 's/--threads//g' $f
     sed -i 's/-fuse-ld=gold//g' $f
     sed -i 's/--thread-count=4//g' $f
-    if [[ "$f" != "out.gn/obj/v8_libbase.ninja" ]]; then
+    if [[ "$f" != "out.gn/obj/v8_libbase.ninja" ]]; then
       if [[ "$f" != "out.gn/obj/third_party/zlib/zlib.ninja" ]]; then
-        sed -ie "s/libs =/libs = -lz/g" $f
+        sed -i "s/libs =/libs = -lz/g" $f
       fi
     fi
   done
   for f in "out.gn/obj/mksnapshot.ninja out.gn/obj/v8.ninja out.gn/obj/wee8.ninja out.gn/obj/d8.ninja"; do
-    sed -ie "s/libs = -latomic/libs = -lz -latomic/g" $f
+    sed -i "s/libs = -latomic/libs = -lz -latomic/g" $f
   done
 
   # [[nodiscard]] support in GCC 9 is not as good as in clang
-  sed -ie "s/# define V8_HAS_CPP_ATTRIBUTE_NODISCARD (V8_HAS_CPP_ATTRIBUTE(nodiscard))//g" include/v8config.h
+  sed -i "s/# define V8_HAS_CPP_ATTRIBUTE_NODISCARD (V8_HAS_CPP_ATTRIBUTE(nodiscard))//g" include/v8config.h
 fi
 
+find out.gn -type f -name '*.ninja' -exec sed -i "
+    s|-Werror||g
+    s|^\s*ldflags = |&${LDFLAGS} |
```

Comment:
We could also inject the other flags like
```suggestion
    s|-Werror||g
    s|^\s*defines = |&${CPPFLAGS/-O2/} |
    s|^\s*cflags = |&${CFLAGS/-O2/} |
    s|^\s*cflags_cc = |&${CXXFLAGS/-O2/} |
    s|^\s*ldflags = |&${LDFLAGS} |
```
(`-O2` omitted since GCC encounters an ICE).
But overall it doesn't feel like the right place to do this... I'd only go with `LDFLAGS` for now.

Reply:
Hmm, so back in https://github.com/conda-forge/libv8-feedstock/runs/8855675833 we had linux64 passing and everything else failing. Now we have the opposite :-). Do we know what changed that broke the linux64 build?

Reply:
See https://github.com/conda-forge/libv8-feedstock/pull/80#pullrequestreview-1147200934 .
Fixing one thing led to revealing a bug in the build script.
TBH, there are multiple issues with this build script and due to lack of explanatory comments/commit messages it does not by itself make its intentions clear. (The script could be made more robust overall -- I'd be willing to do that, but only in cooperation with the author due to the current lack of comments in the script.)